### PR TITLE
Add CDK

### PR DESCRIPTION
**Pull request by kolomied**
https://github.com/kolomied/awesome-cdk

AWS CDK is a popular framework for defining cloud infrastructure in code. There is a vibrant community around the framework that builds solutions on top of it. Having a curated awesome list that mentions the valuable projects would be very helpful.

Reviewed #1832,  #1823

### CODE REVIEW

**Comment by kolomied** [comment]
unicorn

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by kolomied** [comment]
@sindresorhus Went through the checklist again:

- Passed everything through awesome-lint
- Added missing descriptions
- Header cleanup
- Made logo right aligned

Any clues about missing pieces?

**Comment by kolomied** [comment]
@sindresorhus any chance you can review updated repo please?

**Comment by anksos** [comment]
@kolomied I went through your list and haven't noticed anything, majority of the stuff from your initial pr have been fixed. Descriptions some of them are self explanatory so i don't think you had to "over" explain them but I guess is good that you did it anyway.

**Comment by sindresorhus** [comment]
The title used throughout should be `AWS Cloud Development Kit`. Just `CDK` is not very useful.

File: readme.md [hunk 1]
```diff
@@ -301,6 +301,7 @@
 - [Vapor](https://github.com/Cellane/awesome-vapor#readme) - Server-side development in Swift.
 - [Dash](https://github.com/ucg8j/awesome-dash#readme) - Python web app framework.
 - [FastAPI](https://github.com/mjhea0/awesome-fastapi#readme) - Python web app framework.
+- [CDK](https://github.com/eladb/awesome-cdk) - Open-source software development framework for defining cloud infrastructure in code.
```
Comment (sindresorhus) [inline readme.md:304]:
```suggestion
- [CDK](https://github.com/eladb/awesome-cdk#readme) - Open-source software development framework for defining cloud infrastructure in code.
```

**Comment by kolomied** [comment]
@sindresorhus Thanks for the review - I accepted your #readme suggestion.

About `AWS Cloud Development Kit` vs `CDK` - I tend to disagree. The acronym is widely used in community, Google searches, etc. I would say that `CDK` is far more recognizable. 

Less importantly - `AWS Cloud Development Kit` is quite mouthful to be awesome.

**Comment by kolomied** [comment]
@sindresorhus Could you please reconsider the renaming requirement? Any chance we can proceed with this PR?

**Comment by n370** [comment]
Hey @kolomied I believe you can leave https://github.com/kolomied/awesome-cdk#contribute off the body of the content and rather use a `pull_request_template.md` that calls it up to attention at the appropriate moment when someone decide to really make a contribution. I know this has been automatically generated by dar5hak/generator-awesome-list but I'm recommending it anyway.

**Comment by kolomied** [comment]
@n370 Thanks for the suggestion - this is a great idea. Created `pull_request_template.md` and fixed the name of **Contributing** section to adhere to the latest `awesome-lint` release.

@sindresorhus Can you give it another go? I do think the repo is in a good shape now.

**Comment by kolomied** [comment]
@sindresorhus Happy New Year! 

Could you please take a look at the [awesome-cdk](https://github.com/kolomied/awesome-cdk) repository and provide some feedback if you think that this PR still shouldn't be merged?

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1347088862158352389

**Pull request merged**